src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -234,4 +242,27 @@ internal class RecommendationService(
     }
     return null
   }
+
+  fun extendedSentenceDetails(convictionResponse: ConvictionResponse): Pair<String, String> {
+    return if (convictionResponse.sentenceDescription.equals("Extended Determinate Sentence") ||
+      convictionResponse.sentenceDescription.equals("CJA - Extended Sentence")
+    ) {
+      Pair(
+        convictionResponse.sentenceOriginalLength.toString() + " " + convictionResponse.sentenceOriginalLengthUnits,
+        convictionResponse.sentenceSecondLength.toString() + " " + convictionResponse.sentenceSecondLengthUnits
+      )
+    } else Pair("", "")
+  }
+
+  private fun fetchMappaCategory(riskResponse: RiskResponse?): String? {
```

Comment:
Shouldn't you remove this now it appears in the PartATemplateReplacementService.kt class?

Reply:
er ... neat point ... done!